Core/clim-basic/extended-streams/stream-input.lisp

```diff
@@ -169,10 +172,10 @@ keys read."))
         (call-next-method stream last-deadie-gesture))
       (call-next-method)))
 
-(defclass standard-extended-input-stream (extended-input-stream
-                                          ;; FIXME: is this still needed?
-                                          standard-sheet-input-mixin
-                                          dead-key-merging-mixin)
+(defclass standard-extended-input-stream (input-kernel-mixin
+                                          extended-input-stream
+                                          dead-key-merging-mixin
+                                          standard-sheet-input-mixin)
```

Comment:
Doesn't this remove the `stream` superclass (and related stream classes)?

Reply:
I see, the new protocol class takes care of that. But the stream used to be `fundamental-character-input-stream`. Now it is just a `fundamental-input-stream`. Is that intentional?

Reply:
yes, standard extended input stream is not specified to be a character stream, especially because it may contain `event` instances.